skylib/kustomize/kustomize.bzl

```diff
@@ -405,6 +409,10 @@ def _remove_prefixes(s, prefixes):
         s = _remove_prefix(s, prefix)
     return s
 
+def _remove_gitops_suffix(s):
+    s = s.replace("_gitops.yaml", ".yaml")
```

Comment:
this replacement string patterns are so they don't rewrite all of our gitops targets, there might be a better way to do this, but this is the best I could come up with for now